pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -1,34 +1,60 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, cereal, openmp
-, libjpeg ? null
-, zlib ? null
-, libpng ? null
-, eigen ? null
-, libtiff ? null
+{ lib, stdenv, fetchFromGitHub, pkg-config, cmake
+, cereal
 , ceres-solver
+, clp
+, coin-utils
+, eigen
+, flann
+, lemon-graph
+, libjpeg
+, libpng
+, libtiff
+, openmp
+, osi
+, zlib
 , enableShared ? !stdenv.hostPlatform.isStatic
 , enableExamples ? false
 , enableDocs ? false }:
 
 stdenv.mkDerivation rec {
-  version = "unstable-2022-12-30";
+  version = "unstable-2023-09-16";
   pname = "openmvg";
 
   src = fetchFromGitHub {
     owner = "openmvg";
     repo = "openmvg";
-    rev = "e1bbfe801986cd7171f36443a1573b0f69f3702d";
-    sha256 = "sha256-DngfmejNFw5pogTo7Ec5aUey2LUQIojvJybLmtCfvVY=";
+    rev = "d67601b7a70b28f95d444893fe6420fd4cce232e";
+    sha256 = "sha256-jyrrj1d07U2LyA+NDN+SJ0eM+qIRsqKvHi+EF69m2Es=";
     fetchSubmodules = true;
```

Comment:
Since we are no longer using vendored dependencies, we don't need the submodules right?